.github/templates/main.yml

```diff
@@ -187,12 +187,14 @@ steps:
 
 #@ def publishTestsResults(files, test_title):
 name: Publish Unit Test Results
-uses: EnricoMi/publish-unit-test-result-action/composite@v1
+uses: dorny/test-reporter@v1
```

Comment:
Would it make sense to fork it if it's a fixed version anyway to make sure this doesn't start failing if the repo doesn't exist anymore?

Reply:
That's not a fixed version. It'll take any 1.x.x as soon as it's updated. Hence, forking that would mean we manually integrate all the updates. I think that forking that makes sense only the moment that it disappears, with say something like wayback machine. If that would not be possible, there surely are at least 3 to 4 more gh actions for that specific  task. And lastly, I'm sure someone else has already forked it and will likely become the "new de-facto" replacement to the one taken down.

Reply:
I don't think we should be forking 3rd party actions. However, we should fix the commit sha rather than the version to protect against a malicious user taking over the 3rd party repo.

Reply:
> 
> 
> That's not a fixed version. It'll take any 1.x.x as soon as it's updated. Hence, forking that would mean we manually integrate all the updates. I think that forking that makes sense only the moment that it disappears, with say something like wayback machine. If that would not be possible, there surely are at least 3 to 4 more gh actions for that specific task. And lastly, I'm sure someone else has already forked it and will likely become the "new de-facto" replacement to the one taken down.

Totally agree. I only brought it up because it happened to a project I've been part of in the past and it happened during a time where we had a lot of other stress going on when all of a sudden a library that we used to use disappeared. That's where we started to fork things we use to make sure we don't loose dependencies the day before the release. :D